### ISSUE TITLE
More embed query params like "?view=editor" added to the main sandbox editor

### ISSUE THREAD

**Issue by jasdeepkhalsa** [codesandbox/codesandbox-client#6328]
# 🌈 More embed query params like "?view=editor" added to the main sandbox editor

It would be extremely useful to share a sandbox URL with a guest / collaborator that:

* Starts with only the code editor visible and preview pane collapsed e.g. [https://codesandbox.io/s/crazy-spence-c25y6?`view=editor`](https://codesandbox.io/s/crazy-spence-c25y6?view=editor)
* Starts with multiple tabs already open which I've specified, the settings sidebar hidden (this is a new query param I am suggesting for both the embed and the main sandbox editor) and the preview pane is collapsed too e.g. [https://codesandbox.io/s/crazy-spence-c25y6?`view=editor`&`module=/package.json,/index.html`&`hidesidebar=1`](https://codesandbox.io/s/crazy-spence-c25y6?view=editor&module=/package.json,/index.html&hidesidebar=1)

In summary, I believe the following query params (in order of priority) would be really useful for the main sandbox editor, borrowed from the embed editor:

* `view` - `editor`, `split` (default) or `preview`
* `hidesidebar` (this is a new query param I am suggesting for both the embed and the main sandbox editor) - `0` (default) or `1`
* `module` - paths to module(s) (starting with `/` and comma separated) or entry path (default)

Please thumbs up this feature if you would like to see it get implemented!

**Comment by JamesACS** [comment]
Thanks for another detailed feature request @jasdeepkhalsa - I've added this to our internal board as well

**Comment by EricMcWinNer** [comment]
It would be awesome if you could also specify the default preview url for the embedded view. That way you can embed the same repository several times on a single page with different starting preview urls

**Comment by skf-funzt** [comment]
It seems that the embeds are stripped down in case of configuration.
None of the query parameters are working, tried several ones from the [SanboxEmbed.tsx](https://github.com/codesandbox/codesandbox-client/blob/master/packages/react-embed/src/SandboxEmbed/SandboxEmbed.tsx) but with no luck.

For me, in the current state, the embeds are quite unattractive to use.
